docs/Webhooks/Creating_subscription.rst

```diff
@@ -0,0 +1,55 @@
+.. _creating_subscription:
+
+Creating webhook subscription (Beta version)
+============================================
+
+To create a webhook subscription, send a ``POST`` request to ``/webhooks``.
+The body needs to contain the following JSON data. For example:
+
+  .. code-block:: JSON
+   :linenos:
+    {
+        "description": "Subscription for events",
+        "url": "https://ennpjtc2q1cm.x.pipedream.net",
+        "events": [
+            "OrderCreated"
+        ]
+    }
+
+The response, if successfully will be a JSON object with the following data:
+
+  .. code-block:: JSON
+   :linenos:
+
+    {
+        "success": true
+    }
+
+If you can no longer create a webhook subscription, you will receive a ``500 Internal Server Error``
+
+***********
+Event Types
+***********
+
+The list of event types that you can subscribe to recieve real-time updates.
+
+.. csv-table::
+   :header: "Event Type", "Description"
+   :widths: 30,70
+
+   "``OrderCreated``","This type will notify for order created event"
+   "``LineItemStateTransition``","This type will notify when an orderline's status is changed :ref:`See Orderline status for more information.<_orderline_status>`"
```

Comment:
Would be nice to have some sample payloads for event as well? As a dev I would want to know how the subscribed event would look like.